mozilla-release/mobile/android/chrome/content/aboutAddons.js

```diff
@@ -242,6 +242,17 @@ var Addons = {
 
     return outer;
   },
+  /* Cliqz start */
+  // add check for should hidden addons
+  shouldHidden: function(addonId) {
```

Comment:
`shouldHidden` is not English, `shouldBeHidden` or `shouldHide`.
Is the method meant to be used internally only (private)? If so, prepend an underscore (i.e. `_shoudlBeHidden`).